### ISSUE TITLE
TypeError: Modularizer.installMultipleInternalModules is not a function

### ISSUE THREAD

**Issue by omjokine** [Unitech/pm2#3901]
## What's going wrong?
> pm2 install typescript
command produces error:
```
[PM2][Module] Installing module typescript
/src/node_modules/pm2/lib/API/Modules/Modularizer.js:85
      Modularizer.installMultipleInternalModules(currentModule.dependencies, cb);
                  ^

TypeError: Modularizer.installMultipleInternalModules is not a function
    at Object.Modularizer.install (/src/node_modules/pm2/lib/API/Modules/Modularizer.js:85:19)
    at API.CLI.install (/src/node_modules/pm2/lib/API/Modules/Modules.js:92:17)
    at Command.<anonymous> (/src/node_modules/pm2/bin/pm2:493:9)
    at Command.listener (/src/node_modules/pm2/node_modules/commander/index.js:315:8)
    at Command.emit (events.js:182:13)
    at Command.parseArgs (//src/node_modules/pm2/node_modules/commander/index.js:651:12)
    at Command.parse (/src/node_modules/pm2/node_modules/commander/index.js:474:21)
    at beginCommandProcessing (/src/node_modules/pm2/bin/pm2:159:13)
    at /src/node_modules/pm2/bin/pm2:233:7
    at /src/node_modules/pm2/lib/API.js:178:16
```

## How could we reproduce this issue?
Install latest pm2 (3.1.0) with command
> npm install pm2
and then
> pm2 install typescript

## Supporting information
pm2 version 2.10.4 works as expected

**Comment by wallet77** [comment]
@omjokine 

I got it, I work on it, we will release as soon as possible.

**Comment by wallet77** [comment]
Fixed in 3.1.1

**Comment by omjokine** [comment]
wow that was fast :open_mouth: now it is working as expected. thanks @wallet77 !
